src/_help/documentation-experience/api-explorer.md

```diff
@@ -0,0 +1,72 @@
+---
+title: API Explorer (Beta)
+---
+
+- TOC
+{:toc}
+
+The API Explorer is a new feature in beta with limited access. You can request access by following this link.
+
+## What is the API Explorer?
+
+The API Explorer allows you to test an API in real-world conditions directly from its documentation. It facilitates API discovery by enabling you to try out specific API calls on any server listed in your definition file. It helps you get started quickly without a testing environment or share a specific use case for discovery, support, or debugging purposes.
+
+## How to use the API Explorer?
+
+The API Explorer is accessible at any time via a button at the top of your documentation (and remains visible as you scroll). Each operation also features a button that opens the API Explorer for that specific operation.
+
+[screenshot of the button]
+
+If you haven’t opened the API Explorer from a specific operation, you can select one from the corresponding menu.
+
+[screenshot of the API Explorer]
+
+We identify the required fields directly from the definition file, making it easier for you to fill out the request, detecting whether it’s a boolean, date, etc. Fill in the expected information to execute the request and receive a response.
+
+[screenshot of a response]
+
+### Sharing
+
+It’s possible to share a request setup, which is useful for showing an example in a demo or for support purposes. The share button generates a URL that includes the pre-filled request parameters.
+
+> This URL will never share your authentication parameters or the response.
+{: .info}
+
+[screenshot of sharing]
+
+If the API documentation has been updated after the share URL was generated, you will be notified when attempting to use it. You will then have the option to switch to an updated version of the operation. Note that in this case, new elements will no longer be pre-filled.
```

Comment:
You won't have the option to switch to an updated version.

The version of the API Explorer you'll see is based on the last deployment.

But, for example, if the request was generated 1 month before, and a new version was deployed since, you'll get a message at the top of your API Explorer, saying that properties may have changed, and linking you to the right compare page.
<img width="987" alt="image" src="https://github.com/user-attachments/assets/9298b0d0-5f7c-4eab-86d3-5f4bee6c08bf">

Reply:
Thank you for the clarification; your comments on the previous draft were indeed a bit unclear. I’ll make the necessary changes.